cadence.go

```diff
@@ -49,42 +48,34 @@ func (e englishPattern) ToCrontab() string {
 }
 
 func parseEnglishPattern(pattern string) (*englishPattern, error) {
-	orig := pattern
-	pattern = strings.TrimSpace(pattern)
-	pattern = strings.ToLower(pattern)
-
-	if !strings.HasPrefix(pattern, "every") {
-		return nil, fmt.Errorf("invalid prefix for pattern `%s`", orig)
-	}
-
-	// there should be a number as the next character.
-	pattern = strings.TrimPrefix(pattern, "every")
-	pattern = strings.TrimSpace(pattern)
-
-	var bld strings.Builder
-
-	for _, r := range pattern {
-		if unicode.IsDigit(r) {
-			bld.WriteRune(r)
+	var tokens []string = strings.Fields(pattern)
```

Comment:
this is a complete rewrite based on splitting the pattern into tokens a priori and working with them. I think it's easier to reason about what's happening that way and some logic that is implicit in the old version, becomes more implicit this way. E.g. rejecting a string if it has more than maximum 3 tokens.

Reply:
No need to declare the type here!

```suggestion
	tokens := strings.Fields(pattern)
```

Otherwise, didn't know about `strings.Fields`, decent addition. If we have more complex lexing rules we'll have to remove this anyway so this works great for now.